pages/facts.html

```diff
@@ -4,9 +4,59 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Cold Hard Facts</title>
+  <link rel="stylesheet" href="../stylesheets/style.css">
+  <link rel="stylesheet" href="../stylesheets/facts.css">
 </head>
 <body>
-  
+  <header>
+    <h1>Cold Hard Facts about the Blue Ridge Mountains</h1>
+    <nav id="nav">
+      <ul>
+        <li>
+          <a href="../pages/index.html">Home</a>
+        </li>
+        <li>
```

Comment:
Some list elements have the anchor tag inline and some don't, to make things easy to read and reduce possible confusion, we should pick one and be consistent across the project.